admin_guide/garbage_collection.adoc

```diff
@@ -15,14 +15,20 @@ toc::[]
 The {product-title} node performs two types of garbage collection:
 
 * xref:container-garbage-collection[Container garbage collection]: Removes
-terminated containers. Typically run every minute.
+terminated containers.
 * xref:image-garbage-collection[Image garbage collection]: Removes images not
-referenced by any running pods. Typically run every five minutes.
+referenced by any running pods.
 
 [[container-garbage-collection]]
-
 == Container Garbage Collection
 
+Container garbage collection is enabled by default and happens automatically in
+response to eviction thresholds being reached. The node tries to keep any pod
+that is still accessible from the API. If the pod has been deleted, the
+containers will be as well. Containers are only deleted when the pod object is
+gone from {product-title}. If the node is under disk pressure, it will remove
```

Comment:
I would say "Containers are preserved as long the pod is not deleted and there an eviction threshold is not reached."